src/Validation/ValidationCountsRestController.php

```diff
@@ -49,6 +49,8 @@ public function __construct( UserAccess $user_access ) {
 		$this->namespace             = 'amp/v1';
 		$this->rest_base             = 'unreviewed-validation-counts';
 		$this->dev_tools_user_access = $user_access;
+
+		add_filter( 'posts_where', [ AMP_Validation_Error_Taxonomy::class, 'filter_posts_where_for_validation_error_status' ], 10, 2 );
```

Comment:
The filter above was previously not running and so the new validation count was instead returning the count for all validated posts.

Putting the filter to run when this service is being constructed fixes the issue, but I'm not sure if it's the best approach.

Reply:
Actually, shouldn't it go into `\AMP_Validated_URL_Post_Type::get_validation_error_urls_count()`? That method seems to be broken. It is supposed to be returning the count of validation errors that have unreviewed errors, but it is returning everything. I see currently it's because the `posts_where` filter is only being added in `add_admin_hooks()`  which runs if `is_admin()`. Since the REST API is not in the admin, this is why the filters aren't added.